.gitlab/build/force_build_gcs_image.yml

```diff
@@ -5,9 +5,11 @@ stages:
 build-gcs:
   stage: build
   variables:
-    IMAGE_TAG: "datafed/gcs"
+    PROJECT: "datafed"
+    COMPONENT: "gcs"
     GIT_STRATEGY: clone
     DATAFED_HARBOR_REGISTRY: "$REGISTRY" # needed by c_harbor_artifact_count
```

Comment:
What is this comment??? Is that meant to be ./scripts/ci_harbor_artifact_count.sh